presto-main/src/main/java/com/facebook/presto/operator/aggregation/GenericAccumulatorFactory.java

```diff
@@ -748,6 +762,11 @@ public GroupByIdBlock getGroupByIdBlock()
             {
                 return groupByIdBlock;
             }
+
+            public long getRetainedSizeInBytes()
+            {
+                return groupByIdBlock.getRetainedSizeInBytes() + page.getRetainedSizeInBytes();
```

Comment:
also add the `INSTANCE_SIZE` for `GroupIdPage` itself: `ClassLayout.parseClass(GroupIdPage.class).instanceSize();`